deploy/charts/alluxio/templates/helpers/_alluxio-env.tpl

```diff
@@ -0,0 +1,58 @@
+{{- /* ===================================== */}}
```

Comment:
should this file have the license header? same comment for other .tpl files

Reply:
Good question. I noticed that in Alluxio/alluxio repo we didn't include a header in the .tpl file, so I kept as it was. I think we should include headers in those files too.